src/Hero6.Tests/Engine/Utilities/Settings/UserSettingsTests.cs

```diff
@@ -4,6 +4,8 @@
 // 'LICENSE.CODE.md', which is a part of this source code package.
 // </copyright>
 
+using NSubstitute;
```

Comment:
This produces a warning 🙃

Reply:
It's much less common to put usings inside the NS, but that does appear to be style in use on this project. Fixed.